packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -112,50 +73,195 @@ const requiredEnvVars: RequiredEnvVars = {
  * SEQUENCER_HD_PATH
  * PROPOSER_HD_PATH
  */
-const env = process.env
-const FRAUD_SUBMISSION_ADDRESS = env.FRAUD_SUBMISSION_ADDRESS || 'no fraud'
-const DISABLE_QUEUE_BATCH_APPEND = !!env.DISABLE_QUEUE_BATCH_APPEND
-const MIN_GAS_PRICE_IN_GWEI = parseInt(env.MIN_GAS_PRICE_IN_GWEI, 10) || 0
-const MAX_GAS_PRICE_IN_GWEI = parseInt(env.MAX_GAS_PRICE_IN_GWEI, 10) || 70
-const GAS_RETRY_INCREMENT = parseInt(env.GAS_RETRY_INCREMENT, 10) || 5
-const GAS_THRESHOLD_IN_GWEI = parseInt(env.GAS_THRESHOLD_IN_GWEI, 10) || 100
-
-// Private keys & mnemonics
-const SEQUENCER_PRIVATE_KEY = env.SEQUENCER_PRIVATE_KEY
-const PROPOSER_PRIVATE_KEY =
-  env.PROPOSER_PRIVATE_KEY || env.SEQUENCER_PRIVATE_KEY // Kept for backwards compatibility
-const SEQUENCER_MNEMONIC = env.SEQUENCER_MNEMONIC || env.MNEMONIC
-const PROPOSER_MNEMONIC = env.PROPOSER_MNEMONIC || env.MNEMONIC
-const SEQUENCER_HD_PATH = env.SEQUENCER_HD_PATH || env.HD_PATH
-const PROPOSER_HD_PATH = env.PROPOSER_HD_PATH || env.HD_PATH
-// Auto fix batch options -- TODO: Remove this very hacky config
-const AUTO_FIX_BATCH_OPTIONS_CONF = env.AUTO_FIX_BATCH_OPTIONS_CONF
-const autoFixBatchOptions: AutoFixBatchOptions = {
-  fixDoublePlayedDeposits: AUTO_FIX_BATCH_OPTIONS_CONF
-    ? AUTO_FIX_BATCH_OPTIONS_CONF.includes('fixDoublePlayedDeposits')
-    : false,
-  fixMonotonicity: AUTO_FIX_BATCH_OPTIONS_CONF
-    ? AUTO_FIX_BATCH_OPTIONS_CONF.includes('fixMonotonicity')
-    : false,
-  fixSkippedDeposits: AUTO_FIX_BATCH_OPTIONS_CONF
-    ? AUTO_FIX_BATCH_OPTIONS_CONF.includes('fixSkippedDeposits')
-    : false,
-}
 
 export const run = async () => {
+  dotenv.config()
+
+  const config: Bcfg = new Config('batch-submitter')
+  config.load({
+    env: true,
+    argv: true,
+  })
+
+  // Parse config
+  const env = process.env
+  const environment = config.str('node-env', env.NODE_ENV)
+  const network = config.str('eth-network-name', env.ETH_NETWORK_NAME)
+  const release = `batch-submitter@${env.npm_package_version}`
+  const sentryDsn = config.str('sentry-dsn', env.SENTRY_DSN)
+  const sentryTraceRate = config.ufloat(
+    'sentry-trace-rate',
+    parseInt(env.SENTRY_TRACE_RATE, 10) || 0.05
+  )
+
+  /* Logger */
+  const name = 'oe:batch_submitter:init'
+  let logger
+
+  if (config.bool('use-sentry')) {
```

Comment:
to confirm, this is the only new variable added / changed in this PR?

Reply:
Yes this is the only new variable added - it is used to be explicit and we should make sure that we use the same name across the different services

Reply:
@tynes this defaults to false then i'm assuming? would appreciate adding this to `.env.example`!